lib/extensions/iex_client.rb

```diff
@@ -15,6 +15,12 @@ def iex_meta
         iex.get('/account/metadata', token: @iex_tokens[:secret])
       end
 
+      def stonk_data(stonk, extended: false)
+        pos = stonk.send(stonk_method(:change_percent, extended)).positive?
```

Comment:
`.positive?` will be a no method error

Reply:
technically probably wouldn't get here if its not extended hours, but good call, updated to make it null-safe.